Many Types Make Light Work.md

```diff
@@ -12,47 +12,29 @@
 
 ---
 
-# Reusing code
+# More code = more complexity = more bugs
```

Comment:
I understand the point you're making here, but I don't think more code _always_ means more complexity. Sometimes you add code to reduce complexity.

I'm not sure if that's too nit-picky or nuanced to be helpful…